webapp/components/Notification/Notification.vue

```diff
@@ -4,8 +4,8 @@
       <ds-space margin-bottom="x-small">
         <user-teaser :user="from.author" :date-time="from.createdAt" />
       </ds-space>
-      <ds-text class="reason-text-for-test" color="soft">
-        {{ $t(`notifications.reason.${notification.reason}`) }}
+      <ds-text data-test="reason-text" color="soft">
+        {{ $t(notificationReason) }}
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-13T08:54:50Z" title="Thursday, February 13th 2020, 9:54:50 am +01:00">Feb 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,8 +4,8 @@
       <ds-space margin-bottom="x-small">
         <user-teaser :user="from.author" :date-time="from.createdAt" />
       </ds-space>
-      <ds-text class="reason-text-for-test" color="soft">
-        {{ $t(`notifications.reason.${notification.reason}`) }}
+      <ds-text data-test="reason-text" color="soft">
+        {{ $t(notificationReason) }}
```


The decision details which you skipped to implement like I explained above have to be shown here @roschaefer !!!